public-ui/src/app/data-browser/components/survey-version-table/survey-version-table-react.component.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+    AfterViewInit,
+    Component,
+    ElementRef,
+    EventEmitter,
+    Input,
+    Output,
+    ViewChild,
+    ViewEncapsulation
+} from '@angular/core';
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import * as ReactDOM from 'react-dom';
+import { BaseReactWrapper } from '../../../data-browser/base-react/base-react.wrapper';
+import { ClrIcon } from '../../../utils/clr-icon';
+
+const containerElementName = 'root';
+
+interface Props {
+    surveys: any;
+}
+
+export class SurveyVersionTableReactComponent extends React.Component<Props, {}> {
+constructor(props: Props) {
+    super(props);
+    this.processSurveys();
+}
+
+processSurveys() {
+    this.props.surveys.sort((a1, a2) => {
+        if (a1.monthNum.split('/')[0] < a2.monthNum.split('/')[0]) {
+            return -1;
+        }
+        if (a1.monthNum.split('/')[0] > a2.monthNum.split('/')[0]) {
+            return 1;
+        }
+        return 0;
+    });
+this.props.surveys.forEach((survey, i) => {
+    survey['pdfLink'] = '/assets/surveys/' + survey.monthName.replace('/', '_') + '_COPE_COVID_English_Explorer.pdf';
+});
+}
+
+render() {
+    return <div className='version-box-container'>
+            <h5><strong>Survey versions</strong></h5>
+                <br />
+            <div className='version-box'>
+                <div className='version-box-header'>
+                <div className='version-box-item'>Month</div>
+                <div className='version-box-item'>Participants</div>
+                <div className='version-box-item'>Number of Questions</div>
+                <div className='version-box-item'>Download PDF</div>
+            </div>
+            <div className='version-box-body'>
+            {
+                this.props.surveys.map((survey, index) => {
```

Comment:
if the above changes are made, we'll need to add condition here to only render after the surveys are set:
```suggestion
                !!this.state.surveys && this.state.surveys.map((survey, index) => {
```

Reply:
Nit: remove unused `index`